libcxx/test/std/utilities/meta/meta.rel/is_virtual_base_of.pass.cpp

```diff
@@ -9,7 +9,7 @@
 // UNSUPPORTED: c++03, c++11, c++14, c++17, c++20, c++23
 
 // These compilers don't support __builtin_is_virtual_base_of yet.
-// UNSUPPORTED: clang-18, clang-19, gcc-14, apple-clang-16, apple-clang-17
+// UNSUPPORTED: clang-19, gcc-14, apple-clang-16, apple-clang-17
```

Comment:
If after: https://github.com/llvm/llvm-project/pull/131883 we have Apple Clang 17 (Xcode 16.3). Can we also get rid of:
```suggestion
// UNSUPPORTED: clang-19, gcc-14,apple-clang-17
```

Reply:
Also these need to be removed:
https://github.com/llvm/llvm-project/blob/48441cb8a2fa3b3f9502ba4ba1242746615841cb/libcxx/test/std/utilities/variant/variant.visit.member/robust_against_adl.pass.cpp#L10
https://github.com/llvm/llvm-project/blob/48441cb8a2fa3b3f9502ba4ba1242746615841cb/libcxx/test/std/utilities/variant/variant.visit.member/visit.pass.cpp#L11
https://github.com/llvm/llvm-project/blob/48441cb8a2fa3b3f9502ba4ba1242746615841cb/libcxx/test/std/utilities/variant/variant.visit.member/visit_return_type.pass.cpp#L11
https://github.com/llvm/llvm-project/blob/48441cb8a2fa3b3f9502ba4ba1242746615841cb/libcxx/test/std/utilities/format/format.arguments/format.arg/visit.pass.cpp#L12
https://github.com/llvm/llvm-project/blob/48441cb8a2fa3b3f9502ba4ba1242746615841cb/libcxx/test/std/utilities/format/format.arguments/format.arg/visit.return_type.pass.cpp#L12
https://github.com/llvm/llvm-project/blob/48441cb8a2fa3b3f9502ba4ba1242746615841cb/libcxx/test/std/utilities/format/format.arguments/format.arg/visit_format_arg.deprecated.verify.cpp#L11

Reply:
I removed all Apple-Clang-15 and Apple-Clang-16 references in the next patch and addressed the XFAILs